shared/redux/rtk-apis/messages/messages.api.ts

```diff
@@ -9,9 +9,18 @@ const messagesApi = projectApi.injectEndpoints({
         method: "POST",
         body: messageData,
       }),
+      invalidatesTags: ["Messages"],
+    }),
+
+    getClassroomMessages: builder.query<IMessage[], number>({
+      query: (classroomId) => ({
+        url: `/messages/classroom/${classroomId}`,
```

Comment:
Please follow REST principles. The endpoint should be: `classroooms/:classroomId/messages`